### DISCUSSION TITLE
Netbox on Red Hat linux

### DISCUSSION THREAD

**Discussion by berkeleycowboy** [netbox-community/netbox#12854] [Q&A]
Has anybody successfully been able to get Netbox installed on Red Hat 8.X

My installation issue seems to be associated with the Django installation.  

Thanks in advance for your suport.

**Comment by candlerb** [comment]
> My installation issue seems to be associated with the Django installation.

That won't be an issue if you follow the Netbox installation instructions correctly, because it will install all its dependent python packages (including Django) inside a virtualenv.  There is *no* use of either system python libraries, or globally-installed python libraries ("pip3 install" outside of the virtualenv)

Can you show what actual error you saw, and what command you typed to trigger it?
